File: Python/Maths/Prime.py
import math

if __name__ == '__main__':
    
    b, isPrime = 7, True

    for i in range(2, int(math.sqrt(b)) + 1, 1):
        
        if b % i == 0:
            isPrime = False
            break
        
    print(isPrime)

File: Java/Maths/Prime.java
class Innoskrit {
	public static void main (String[] args) {
		int b = 7;
		boolean isPrime = true;

		for(int i = 2; i < Math.sqrt(b); i++) {
			if(b % i == 0) {
				isPrime = false;
				break;
			}
		}
	    
	    System.out.println(isPrime);
        
	}
}
